english/marshmallow.md

```diff
@@ -968,3 +968,5 @@ Time to [move to Canada](Canada/Canada.md), eh?
 The crushing weight of all these options causes you to [end your life.](end/end.md)
 
 You're reincarnated as a green starfish but wish you were [blue instead.](starfish/starfish.md)
+
+You wake up as Harry Potter ... and are facing Voldemort in a duel? What should you [do](magicaladventure/decision_time/decision_time.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.